diff --git a/dist/floating-vue.mjs b/dist/floating-vue.mjs
index d54beb90f1fd341fb0aa41e1503afdbbece12cbb..6e5f67632a095a2d48639a2b78a4703aa4385197 100644
--- a/dist/floating-vue.mjs
+++ b/dist/floating-vue.mjs
@@ -604,7 +604,7 @@ const q = "__floating-vue__popper", Q = () => O({
     async $_applyShowEffect() {
       if (this.$_hideInProgress)
         return;
-      if (this.computeTransformOrigin) {
+      if (this.computeTransformOrigin && this.$_referenceNode && this.$_popperNode) {
         const t = this.$_referenceNode.getBoundingClientRect(), o = this.$_popperNode.querySelector(".v-popper__wrapper"), i = o.parentNode.getBoundingClientRect(), s = t.x + t.width / 2 - (i.left + o.offsetLeft), r = t.y + t.height / 2 - (i.top + o.offsetTop);
         this.result.transformOrigin = `${s}px ${r}px`;
       }
@@ -714,6 +714,7 @@ const q = "__floating-vue__popper", Q = () => O({
         e ? t.shownChildren.add(this.randomId) : (t.shownChildren.delete(this.randomId), t.pendingHide && t.hide()), t = t.parentPopper;
     },
     $_isAimingPopper() {
+      if (!this.$_referenceNode || !this.$_popperNode) return !1;
       const e = this.$_referenceNode.getBoundingClientRect();
       if (y >= e.left && y <= e.right && _ >= e.top && _ <= e.bottom) {
         const t = this.$_popperNode.getBoundingClientRect(), o = y - c, i = _ - m, r = t.left + t.width / 2 - c + (t.top + t.height / 2) - m + t.width + t.height, p = c + o * r, a = m + i * r;
